assets/js/modules/analytics-4/datastore/__fixtures__/webdatastreams.json

```diff
@@ -1,24 +1,28 @@
 [
-  {
-    "_id": "2000",
-    "_propertyID": "1000",
-    "name": "properties/1000/webDataStreams/2000",
-    "measurementId": "1A2BCD345E",
-    "firebaseAppId": "",
```

Comment:
As shown in the json in the PR description, `firebaseAppId` is not included here. This is also now part of the `webStreamData` object but not included in the JSON representation when not set.